src/local-ssh/starter.sh

```diff
@@ -0,0 +1 @@
+ELECTRON_RUN_AS_NODE=1 "$@"
```

Comment:
Do we need `#!/usr/bin/env bash` at the top? vscode scripts have them

Reply:
I think so 😄 

<img width="903" alt="image" src="https://github.com/gitpod-io/gitpod-vscode-desktop/assets/29888641/22841d07-b293-4e26-9367-40a1d3b2947f">

Reply:
Good catch 👍 , I checked vscode repo, they are using `#!/usr/bin/env bash` I will switch to use it to align our changes (same with .bat file `setlocal`)